iceoryx_examples/icedelivery/README.md

```diff
@@ -242,25 +227,62 @@ The typed publisher application is an example for a high-level user API and does
 described before. In this summary, just the differences to the prior publisher application are described.
 
 Starting again with the includes, there is now a different one:
+
 ```cpp
 #include "iceoryx_posh/popo/publisher.hpp"
 ```
 
 When it comes to the runtime, things are the same as in the untyped publisher. However, a typed publisher object is
-created
+created and the transmitted data type is provided as template parameter:
+
 ```cpp
 iox::popo::Publisher<RadarObject> publisher({"Radar", "FrontLeft", "Object"});
 ```
 
+The topic type must be default- and copy-constructible when the typed API is used.
+
 A similar while-loop is used to send the data to the subscriber. In contrast to the untyped publisher the typed one
-offers two additional possibilities
+offers three additional possibilities.
+
+#### #2 Functional approach with loaning
```

Comment:
Wrong header: Classic approach with loaning since we have a `if` `else` in the code. 
But the order is correct, please leave it has an alternative and second approach and endorse the `and_then`/`or_else` first!